docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -271,6 +271,32 @@ faster and more stable. Add the ``--headless`` option to enable this mode.
 
 .. _selenium.webdriver: https://github.com/SeleniumHQ/selenium/tree/trunk/py/selenium/webdriver
 
+For testing changes to the admin UI, the selenium tests can be run with the
+``--screenshots`` option enabled. Screenshots will be saved to the
+``tests/screenshots/`` directory.
+
+To define when screenshots should be taken during a selenium test, the test
+class must use the ``django.test.selenium.screenshot_cases`` decorator with a
+list of supported screenshot types (currently ``"desktop"``, ``"mobile"``,
+``"rtl"`` and ``"dark"``). It can then call
```

Comment:
You also have `small_screen` above? Or was the intent to get rid of that?

Reply:
Not sure whether it's worth stating that `rtl` and `dark` only operate with `desktop` size?